server/rest/twofishes.py

```diff
@@ -0,0 +1,115 @@
+        """Creates a shapely geometry from wkt"""
+        return loads(wkt)
+
+    @staticmethod
+    def createGeojson(twofishes, locations):
+        """Create geojson for given locations and twofishes url"""
+        geoms = []
+
+        for i in locations:
+            wkt = TwoFishes.getWktFromTwoFishes(twofishes, i)
+            geom = TwoFishes.createGeometryFromWkt(wkt)
+            for g in geom:
+                geoms.append(geojson.Feature(geometry=g,
+                                             properties={'location': i}))
+
+        multiPoly = geojson.FeatureCollection(geoms)
+
+        return multiPoly
+
+    def createMinervaDataset(self, geojsonString, name):
+        """Creates a dataset from a geojson string"""
+        output = StringIO.StringIO(json.dumps(geojsonString))
+        outputSize = output.len
+        user = self.getCurrentUser()
+        datasetFolder = findDatasetFolder(user, user)
+        itemModel = ModelImporter.model('item')
+        uploadModel = ModelImporter.model('upload')
+        item = itemModel.createItem(name, user, datasetFolder)
+        geojsonFile = uploadModel.uploadFromFile(output, outputSize, name,
+                                                 'item', item, user)
+        GeojsonDataset().createGeojsonDataset(itemId=geojsonFile['itemId'],
+                                              params={})
+        return geojsonFile
+
+    @access.public
+    def autocomplete(self, params):
+        r = requests.get(params['twofishes'],
+                         params={'autocomplete': True,
+                                 'query': params['location'],
+                                 'maxInterpretations': 10,
+                                 'autocompleteBias': None})
+
+        return [i['feature']['matchedName'] for i in r.json()['interpretations']]
+
+    autocomplete.description = (
+        Description('Autocomplete result for a given location name')
+        .param('twofishes', 'Twofishes url')
+        .param('location', 'Location name to autocomplete')
+    )
+
+    @access.public
+    def getGeojson(self, params):
+        locations = json.loads(params['locations'])
+        geojson = TwoFishes.createGeojson(params['twofishes'], locations)
+        return geojson
+
+    getGeojson.description = (
+        Description('Create a geojson string from multiple locations')
+        .param('twofishes', 'Twofishes url')
+        .param('locations', 'List of locations', dataType='list')
+    )
+
+    @access.public
+    def postGeojson(self, params):
+        twofishes = params['twofishes']
+        try:
+            locationInfo = json.loads(params['locations'])
+            geojson = TwoFishes.createGeojson(twofishes, locationInfo)
+        except ValueError:
+            locationInfo = params['locations']
+            geojson = TwoFishes.createGeojson(twofishes, locationInfo)
+
+        minervaDataset = self.createMinervaDataset(geojson, params['name'])
```

Comment:
That endpoint does not have any side effects. It just gets a WKT from twofishes and converts that to geojson. I thought it might be useful since a lot of people work with geojson in BSVE. Also it was good for debugging.